core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -656,7 +661,7 @@ public void connect() {
         // Default move speed
         // Bedrock clients move very fast by default until they get an attribute packet correcting the speed
         attributesPacket.setAttributes(Collections.singletonList(
-                GeyserAttributeType.MOVEMENT_SPEED.getAttribute()));
+            GeyserAttributeType.MOVEMENT_SPEED.getAttribute()));
```

Comment:
i'm sorry, but please change these continuation indent changes. we haven't set a standard for it yet and i would like to reduce unecessary diff in the case we stick with the larger continuation indent

Reply:
I have no idea what is causing this, IntelliJ keeps formatting everything like this and i dont know how to fix it

Reply:
pretty sure you can configure it on a per project basis
check here maybe
https://www.jetbrains.com/help/idea/indentation.html

Reply:
Fixed in latest